<!DOCTYPE html>
<html lang="en">

<head>
    <meta charset="utf-8">
    <meta http-equiv="X-UA-Compatible" content="IE=edge">
    <meta name="viewport" content="width=device-width, initial-scale=1.0">
    <meta name="description" content="Jonathan Marchini research">
    <meta name="author" content="Jonathan Marchini">
    <link rel="canonical" href="https://sourcefoundry.org/cinder/news/">
    <link rel="shortcut icon" href="../img/favicon.ico">

    
    <title>News - jmarchini</title>
    

    <link rel="stylesheet" href="https://use.fontawesome.com/releases/v5.12.0/css/all.css">
    <link rel="stylesheet" href="https://use.fontawesome.com/releases/v5.12.0/css/v4-shims.css">
    <link rel="stylesheet" href="//cdn.jsdelivr.net/npm/hack-font@3.3.0/build/web/hack.min.css">
    <link href='//rsms.me/inter/inter.css' rel='stylesheet' type='text/css'>
    <link href='//fonts.googleapis.com/css?family=Open+Sans:300italic,400italic,700italic,400,300,600,700&subset=latin-ext,latin' rel='stylesheet' type='text/css'>
    <link href="../css/bootstrap-custom.min.css" rel="stylesheet">
    <link href="../css/base.min.css" rel="stylesheet">
    <link href="../css/cinder.min.css" rel="stylesheet">

    
        
        <link rel="stylesheet" href="//cdn.jsdelivr.net/gh/highlightjs/cdn-release@9.18.0/build/styles/github.min.css">
        
    

    <!-- HTML5 shim and Respond.js IE8 support of HTML5 elements and media queries -->
    <!--[if lt IE 9]>
            <script src="https://cdn.jsdelivr.net/npm/html5shiv@3.7.3/dist/html5shiv.min.js"></script>
            <script src="https://cdn.jsdelivr.net/npm/respond.js@1.4.2/dest/respond.min.js"></script>
        <![endif]-->

    

     
</head>

<body>

    <div class="navbar navbar-default navbar-fixed-top" role="navigation">
    <div class="container">

        <!-- Collapsed navigation -->
        <div class="navbar-header">
            <!-- Expander button -->
            <button type="button" class="navbar-toggle" data-toggle="collapse" data-target=".navbar-collapse">
                <span class="sr-only">Toggle navigation</span>
                <span class="icon-bar"></span>
                <span class="icon-bar"></span>
                <span class="icon-bar"></span>
            </button>
            

            <!-- Main title -->

            
              <a class="navbar-brand" href="..">jmarchini</a>
            
        </div>

        <!-- Expanded navigation -->
        <div class="navbar-collapse collapse">
                <!-- Main navigation -->
                <ul class="nav navbar-nav">
                
                
                    <li >
                        <a href="..">Home</a>
                    </li>
                
                
                
                    <li >
                        <a href="../pubs/">Publications</a>
                    </li>
                
                
                
                    <li >
                        <a href="../projects/">Projects</a>
                    </li>
                
                
                
                    <li >
                        <a href="../software/">Software</a>
                    </li>
                
                
                
                    <li >
                        <a href="../people/">People</a>
                    </li>
                
                
                
                    <li >
                        <a href="../teaching/">Teaching</a>
                    </li>
                
                
                
                    <li class="active">
                        <a href="./">News</a>
                    </li>
                
                
                </ul>

            <ul class="nav navbar-nav navbar-right">
                    <li>
                        <a href="#" data-toggle="modal" data-target="#mkdocs_search_modal">
                            <i class="fas fa-search"></i> Search
                        </a>
                    </li>
                    <li >
                        <a rel="prev" href="../teaching/">
                            <i class="fas fa-arrow-left"></i> Previous
                        </a>
                    </li>
                    <li class="disabled">
                        <a rel="next" >
                            Next <i class="fas fa-arrow-right"></i>
                        </a>
                    </li>
            </ul>
        </div>
    </div>
</div>

    <div class="container">
        
        
        <div class="col-md-3"><div class="bs-sidebar hidden-print affix well" role="complementary">
    <ul class="nav bs-sidenav">
        <li class="first-level active"><a href="#10-october-2018">10 October 2018</a></li>
        <li class="first-level "><a href="#29-june-2018">29 June 2018</a></li>
        <li class="first-level "><a href="#6-february-2018">6 February 2018</a></li>
        <li class="first-level "><a href="#17-october-2017">17 October 2017</a></li>
        <li class="first-level "><a href="#21-august-2017">21 August 2017</a></li>
        <li class="first-level "><a href="#21-july-2017">21 July 2017</a></li>
        <li class="first-level "><a href="#10-july-2017">10 July 2017</a></li>
        <li class="first-level "><a href="#29-june-2017">29 June 2017</a></li>
        <li class="first-level "><a href="#15-june-2016">15 June 2016</a></li>
        <li class="first-level "><a href="#10-december-2016">10 December 2016</a></li>
        <li class="first-level "><a href="#20-october-2016">20 October 2016</a></li>
        <li class="first-level "><a href="#1-october-2016">1 October 2016</a></li>
        <li class="first-level "><a href="#22-august-2016">22 August 2016</a></li>
        <li class="first-level "><a href="#1-august-2016">1 August 2016</a></li>
        <li class="first-level "><a href="#26-july-2016">26 July 2016</a></li>
        <li class="first-level "><a href="#30-june-2016">30 June 2016</a></li>
        <li class="first-level "><a href="#13-june-2016">13 June 2016</a></li>
        <li class="first-level "><a href="#6-june-2016">6 June 2016</a></li>
        <li class="first-level "><a href="#31-may-2016">31 May 2016</a></li>
        <li class="first-level "><a href="#5-may-2016">5 May 2016</a></li>
        <li class="first-level "><a href="#15-april-2016">15 April 2016</a></li>
        <li class="first-level "><a href="#10-april-2016">10 April 2016</a></li>
        <li class="first-level "><a href="#7-april-2016">7 April 2016</a></li>
        <li class="first-level "><a href="#6-april-2016">6 April 2016</a></li>
        <li class="first-level "><a href="#7-march-2016">7 March 2016</a></li>
        <li class="first-level "><a href="#28-february-2016">28 February 2016</a></li>
        <li class="first-level "><a href="#23-february-2016">23 February 2016</a></li>
        <li class="first-level "><a href="#23-february-2016_1">23 February 2016</a></li>
        <li class="first-level "><a href="#22-february-2016">22 February 2016</a></li>
        <li class="first-level "><a href="#19-january-2016">19 January 2016</a></li>
        <li class="first-level "><a href="#10-january-2016">10 January 2016</a></li>
        <li class="first-level "><a href="#24-december-2015">24 December 2015</a></li>
        <li class="first-level "><a href="#23-december-2015">23 December 2015</a></li>
        <li class="first-level "><a href="#22-december-2015">22 December 2015</a></li>
        <li class="first-level "><a href="#26-november-2015">26 November 2015</a></li>
        <li class="first-level "><a href="#13-november-2015">13 November 2015</a></li>
        <li class="first-level "><a href="#17-october-2015">17 October 2015</a></li>
        <li class="first-level "><a href="#9-october-2015">9 October 2015</a></li>
        <li class="first-level "><a href="#1-october-2015">1 October 2015</a></li>
        <li class="first-level "><a href="#19-sept-2015">19 Sept 2015</a></li>
        <li class="first-level "><a href="#5-sept-2015">5 Sept 2015</a></li>
        <li class="first-level "><a href="#15-july-2015">15 July 2015</a></li>
        <li class="first-level "><a href="#1-july-2015">1 July 2015</a></li>
        <li class="first-level "><a href="#29-may-2015">29 May 2015</a></li>
        <li class="first-level "><a href="#october-2014">October 2014</a></li>
        <li class="first-level "><a href="#1-august-2014">1 August 2014</a></li>
        <li class="first-level "><a href="#16-june-2014">16 June 2014</a></li>
        <li class="first-level "><a href="#17-apr-2014">17 Apr 2014</a></li>
        <li class="first-level "><a href="#24-mar-2014">24 Mar 2014</a></li>
        <li class="first-level "><a href="#3-oct-2013">3 Oct 2013</a></li>
        <li class="first-level "><a href="#18-sep-2013">18 Sep 2013</a></li>
        <li class="first-level "><a href="#21-mar-2013">21 Mar 2013</a></li>
        <li class="first-level "><a href="#29-dec-2012">29 Dec 2012</a></li>
        <li class="first-level "><a href="#6-dec-2012">6 Dec 2012</a></li>
        <li class="first-level "><a href="#4-dec-2012">4 Dec 2012</a></li>
        <li class="first-level "><a href="#10-nov-2012">10 Nov 2012</a></li>
        <li class="first-level "><a href="#7-nov-2012">7 Nov 2012</a></li>
        <li class="first-level "><a href="#1-november-2012">1 November 2012</a></li>
        <li class="first-level "><a href="#1-november-2012_1">1 November 2012</a></li>
        <li class="first-level "><a href="#1-november-2012_2">1 November 2012</a></li>
        <li class="first-level "><a href="#23-october-2012">23 October 2012</a></li>
        <li class="first-level "><a href="#2-october-2012">2 October 2012</a></li>
        <li class="first-level "><a href="#27-august-2012">27 August 2012</a></li>
        <li class="first-level "><a href="#24-august-2012">24 August 2012</a></li>
        <li class="first-level "><a href="#22-july-2012">22 July 2012</a></li>
        <li class="first-level "><a href="#20-july-2012">20 July 2012</a></li>
        <li class="first-level "><a href="#29-june-2012">29 June 2012</a></li>
        <li class="first-level "><a href="#30-april-2012">30 April 2012</a></li>
        <li class="first-level "><a href="#12-april-2012">12 April 2012</a></li>
        <li class="first-level "><a href="#15-march-2012">15 March 2012</a></li>
    </ul>
</div></div>
        <div class="col-md-9" role="main">

<h4 id="10-october-2018">10 October 2018</h4>
<p>Today we published 2 papers in Nature.</p>
<p><img alt="Screenshot" src="../img/7726.jpeg" /></p>
<p>The first describes our work on the centralized analysis of the genetic data on all 500,000 UK Biobank participants</p>
<p><a href="https://www.nature.com/articles/s41586-018-0571-7">Bycroft et al. (2018) The UK Biobank resource with deep phenotyping and genomic data. Nature 562, pages 203–209.</a></p>
<p>The second describes genome wide association analysis of the brain imaging data collected on ~15,000 UK Biobank participants.</p>
<p><a href="https://www.nature.com/articles/s41586-018-0571-7">Elliott et al. (2018) Genome-wide association studies of brain imaging phenotypes in UK Biobank. Nature 562, pages 210–216.</a></p>
<p>Here is some press that the papers received</p>
<p><a href="https://soundcloud.com/jenny-mills-362181607/uk-biobank-genetics-opens-the-door-to-a-new-era-of-health-research">[Podcast]</a> 
<a href="https://www.wired.co.uk/article/uk-biobank-genetic-studies-healthcare">[Wired article]</a> 
<a href="https://www.nature.com/articles/d41586-018-06950-9">[Nature Editorial]</a> 
<a href="https://www.nature.com/articles/d41586-018-06948-3">[Nature News and Views]</a>
<a href="https://www.news-medical.net/news/20181011/UK-Biobank-provides-wealth-of-information-for-further-genetic-studies.aspx">[News Medical Article]</a></p>
<h4 id="29-june-2018">29 June 2018</h4>
<p>New version of our brain imaging GWAS is now on <a href="https://www.biorxiv.org/content/early/2018/06/29/178806">bioRxiv</a> and includes a larger replication sample, fast multi-trait LMMs using our <a href="https://www.dropbox.com/home/Oxford/software/software_webpages/sbat">SBAT software</a>, co-heritability to brain related diseases and partitioned SNP heritability. …</p>
<h4 id="6-february-2018">6 February 2018</h4>
<p><a href="https://www.dropbox.com/home/Oxford/software/software_webpages/sbat">SBAT software</a> released created by Kevin Sharp. Super-fast multi-trait linear mixed models.</p>
<h4 id="17-october-2017">17 October 2017</h4>
<p>Released today : <a href="http://big.stats.ox.ac.uk/">Oxford Brain Imaging Genetics (BIG) server</a> – a browser for 3,144 GWAS of brain imaging derived phenotypes in the UK Biobank, based on Elliott, L. et al (2017)</p>
<h4 id="21-august-2017">21 August 2017</h4>
<p>Our new Imaging Genetics paper worked on by Lloyd Elliott and Kevin Sharp is just out on bioRxiv</p>
<p><a href="http://www.biorxiv.org/content/early/2017/08/21/178806">Elliott et al. (2017) The genetic basis of human brain structure and function: 1,262 genome-wide associations found from 3,144 GWAS of multimodal brain imaging phenotypes from 9,707 UK Biobank participants.</a></p>
<h4 id="21-july-2017">21 July 2017</h4>
<p>The UK Biobank Genetics paper is now on bioRxiv</p>
<p><a href="http://www.biorxiv.org/content/early/2017/07/20/166298">Bycroft et al. (2017) Genome-wide genetic data on ~500,000 UK Biobank participants.</a></p>
<p><a href="https://www.dropbox.com/sh/k6b34fzw9w4s8bg/AAA65aF5l2oj_AT9iDLgKCv9a?dl=0">IMPUTE 4</a> and <a href="https://jmarchini.org/software/#shapeit-3">SHAPEIT 3</a> released</p>
<h4 id="10-july-2017">10 July 2017</h4>
<p>New version of <a href="https://jmarchini.org/#bgenie/">BGENIE v1.1</a> released. This version improves some of the computational aspects involving threading</p>
<h4 id="29-june-2017">29 June 2017</h4>
<p>Giving a talk today at the <a href="http://www.ukbiobank.ac.uk/2017/06/annual-meeting-live-webcast-2/">UK Biobank Annual Meeting</a></p>
<h4 id="15-june-2016">15 June 2016</h4>
<p>Our new program BGENIE has been released for efficient GWAS for multiple continuous traits and PHEWAS</p>
<h4 id="10-december-2016">10 December 2016</h4>
<p>Giving a keynote talk today at the MLCB workshop at NIPS in Barcelona. My talk is on “Sparse Bayesian Modelling for Multitrait Genetic Association Studies” and will describe research carried out by Kevin Sharp (postdoc in my group) and Valentina Iotchkova (ex-DPhil student in my group).</p>
<h4 id="20-october-2016">20 October 2016</h4>
<p>Our group have the following presentations at ASHG in Vancouver this year</p>
<p>Platform Talk : J. Marchini Thu 20 Oct Session 34. Title : Phasing, imputation and analysis of 500,000 UK individuals genotyped for UK Biobank.</p>
<p>Poster : K.Sharp Thu 20 Oct. Title : A new sparse Bayesian method for multi-trait genome wide association studies</p>
<p>Poster : L. Elliott Fri 21 Oct. Title : Genome-wide association of 2,500 structural and functional brain imaging phenotypes in the UK Biobank study.</p>
<h4 id="1-october-2016">1 October 2016</h4>
<p>Welcome to Zhangyi (Jeffrey) He, Matt Kerin, Simone Rubinacci, Daniel Wells and Chris Gill, who have all started in the group.</p>
<h4 id="22-august-2016">22 August 2016</h4>
<p>The <a href="http://www.haplotype-reference-consortium.org/">Haplotype Reference Consortium</a> paper is now out! Major contributions from Warren Kretzschmar who worked on this project</p>
<p><a href="https://www.nature.com/articles/ng.3643">The Haplotype Reference Consortium. A reference panel of 64,976 haplotypes for genotype imputation. Nature Genetics 10.1038/ng.3643</a></p>
<h4 id="1-august-2016">1 August 2016</h4>
<p>New paper out from Victoria Hore in our group</p>
<p><a href="https://www.nature.com/articles/ng.3624">Victoria Hore, Ana Viñuela, Alfonso Buil, Julian Knight, Mark I McCarthy, Kerrin Small, Jonathan Marchini. Tensor decomposition for multi-tissue gene expression experiments. Nature Genetics 10.1038/ng.3624</a></p>
<p><a href="https://jmarchini.org/software/#sda">SDA (Sparse Decomposition of Arrays) software</a></p>
<h4 id="26-july-2016">26 July 2016</h4>
<p>We have two postdoc positions advertised right now. Get in touch if you want to know more</p>
<p><a href="http://tinyurl.com/zr8l96q">http://tinyurl.com/zr8l96q</a>      <a href="http://tinyurl.com/h36aqrq">http://tinyurl.com/h36aqrq</a></p>
<h4 id="30-june-2016">30 June 2016</h4>
<p>Presentation at The Broad Institute in Boston in their Molecular and Population Genetics seminar series, entitled “Modelling multiple phenotypes : better together”</p>
<h4 id="13-june-2016">13 June 2016</h4>
<p>Giving a talk today at the <a href="http://fsmevents.com/ukbiobank/annual-meeting/">UK Biobank Annual meeting</a>. </p>
<h4 id="6-june-2016">6 June 2016</h4>
<p>New paper out now from our group in Nature Genetics entitled <a href="http://www.nature.com/ng/journal/vaop/ncurrent/full/ng.3583.html">Haplotype estimation for biobank-scale data sets</a> on SHAPEIT3 which is being used to phase the UK Biobank project dataset</p>
<h4 id="31-may-2016">31 May 2016</h4>
<p>Recent interview I did about our <a href="https://wellcome.ac.uk/what-we-do/case-studies/jonathan-marchini">Wellcome Trust Collaborative Award</a>.</p>
<h4 id="5-may-2016">5 May 2016</h4>
<p>Congratulations to Warren Kretzschmar who passed his DPhil Viva today! His thesis title was “Methods for phasing and imputation of very low coverage sequencing data” and was examined by Simon Myers and Heather Cordell.</p>
<h4 id="15-april-2016">15 April 2016</h4>
<p>We have 3 new postdoc positions available to work on methods development for the 100,000 Genomes Project. These posts are funded by a Wellcome Trust Collaborative Award, and will be supervised jointly by Prof Jonathan Marchini, Prof Simon Myers and Dr Garrett Hellenthal (UCL). There are 2 positions in Statistical Genetics Methods Development and 1 post in Statistical Genetics Software Development.</p>
<h4 id="10-april-2016">10 April 2016</h4>
<p>Wellcome to Simone Rubinacci (SysBio DTC) and Daniel Wells (GMS) who have joined the group to do rotation projects.</p>
<h4 id="7-april-2016">7 April 2016</h4>
<p>Presentation at  International Congress of Human Genetics, Kyoto, Japan</p>
<p>Tensor decomposition uncovers trans eQTL networks in the multi-tissue EuroBATS study.</p>
<h4 id="6-april-2016">6 April 2016</h4>
<p>Revised version of the HRC paper now on <a href="https://www.biorxiv.org/content/10.1101/035170v2">bioRxiv</a>, now with replicated low MAF associations found using HRC imputation</p>
<h4 id="7-march-2016">7 March 2016</h4>
<p>Congratulations to Victoria Hore who passed her DPhil Viva today! Her thesis title was “Latent variable models for analysing multidimensional genetic data sets” and was examined by Simon Myers and Oliver Stegle.</p>
<h4 id="28-february-2016">28 February 2016</h4>
<p>Our new paper on phasing for high-coverage sequenced samples using large reference panels has just been published in <a href="http://bioinformatics.oxfordjournals.org/content/early/2016/02/27/bioinformatics.btw065.abstract">Bioinformatics</a></p>
<h4 id="23-february-2016">23 February 2016</h4>
<p>We are pleased to announce that the <a href="http://www.stats.ox.ac.uk/events/probgen16">2nd Probabilistic Modelling in Genomics conference (ProbGen16)</a> will be held at the Department of Statistics in Oxford from 12-14 September 2016. Accommodation will be at Somerville College. We hope this will be an exciting meeting, showcasing much cutting-edge work in the area of Statistical Genomics. </p>
<h4 id="23-february-2016_1">23 February 2016</h4>
<p>We have been awarded a Wellcome Trust Collaborative grant to work on the 100,000 Genomes Project. This work will be jointly supervised with Prof Simon Myers (Oxford) and Dr Garrett Hellenthal (UCL). More details to follow in due course.</p>
<h4 id="22-february-2016">22 February 2016</h4>
<p>Andrew Dahl‘s paper on Bayesian multiple phenotype prediction in genetic studies of related samples is now online at <a href="http://www.nature.com/ng/journal/vaop/ncurrent/full/ng.3513.html">Nature Genetics</a></p>
<p>The method is implemented in an <a href="https://mathgen.stats.ox.ac.uk/genetics_software/phenix/phenix.html">R package called PHENIX</a></p>
<h4 id="19-january-2016">19 January 2016</h4>
<p>Congratulations to Warren Kretzschmar who has just submitted his DPhil thesis!!!</p>
<h4 id="10-january-2016">10 January 2016</h4>
<p>Matt Kerin and Richard Brown have joined the group for their GMS rotation projects.</p>
<h4 id="24-december-2015">24 December 2015</h4>
<p>Haplotype Reference Consortium paper now available on <a href="http://biorxiv.org/content/early/2015/12/23/035170">bioRxiv</a></p>
<h4 id="23-december-2015">23 December 2015</h4>
<p><a href="https://phasingserver.stats.ox.ac.uk/">Oxford Statistics Phasing Server</a> is now live here (still in alphas testing mode)</p>
<p>The server was developed by Winni Kretzschmar and implements a new phasing approach mainly developed by Kevin Sharp</p>
<h4 id="22-december-2015">22 December 2015</h4>
<p>Congratulations to Victoria Hore who has just submitted her DPhil thesis!!!</p>
<h4 id="26-november-2015">26 November 2015</h4>
<p>Presentation at BIOSTATNET workshop on Biomedical (Big) Data at CRM, Barcelona</p>
<p>Tensor decomposition uncovers trans eQTL networks in the multi-tissue EuroBATS study.</p>
<h4 id="13-november-2015">13 November 2015</h4>
<p>We have successfully completed the phasing and imputation of the first ~150,000 samples of the UK Biobank Project. This will be a tremendous resource for the human genetics community over the next decade. We have written a <a href="http://biobank.ctsu.ox.ac.uk/crystal/refer.cgi?id=157020">document</a> which details the statistical analysis involved.</p>
<h4 id="17-october-2015">17 October 2015</h4>
<p>Presentation at Probabilistic Modelling in Genomics 2015 (ProbGen15), CSHL</p>
<p>Modeling high-dimensional phenotypes in genetics—Uncovering brain networks and gene networks</p>
<h4 id="9-october-2015">9 October 2015</h4>
<p>ASHG Platform presentation</p>
<p>Tensor decomposition uncovers trans eQTL networks in the multi-tissue EuroBATS study.
Program Number: 224   Presentation Time: Friday 09 Oct 03:30PM-03:45PM</p>
<h4 id="1-october-2015">1 October 2015</h4>
<p>1000 Genomes Phase 3 paper published in Nature today!</p>
<p>Our group played a central role in the analysis for this project. We designed and implemented the genotype calling and phasing pipeline, which used SHAPEIT2 and MVNcall. Olivier Delaneau, Androniki Menelaou and Warren Kretzschmar all contributed to this work.</p>
<p><a href="http://www.nature.com/nature/journal/v526/n7571/full/nature15393.html">Paper</a><br />
<a href="http://www.nature.com/collections/dcfqmlgsrw">Nature collection</a><br />
<a href="http://www.nature.com/news/variety-of-life-1.18454">Editorial</a><br />
<a href="http://www.nature.com/nature/journal/v526/n7571/full/526052a.html">News and views</a><br />
<a href="http://www.nature.com/nature/podcast/index-2015-10-01.html">Podcast</a>  </p>
<h4 id="19-sept-2015">19 Sept 2015</h4>
<p>The UK10K Project imputation reference panel paper has now been published in <a href="http://www.nature.com/ncomms/2015/150914/ncomms9111/full/ncomms9111.html">Nature Communications</a></p>
<h4 id="5-sept-2015">5 Sept 2015</h4>
<p>Presentation at Genomics of Common Diseases Conference 2015</p>
<p>Title : Tensor decomposition uncovers trans eQTL networks in the multitissue EuroBATS study</p>
<h4 id="15-july-2015">15 July 2015</h4>
<p>The CONVERGE paper is now out in Nature. This study used low-coverage sequencing of 12,000 Chinese women to uncover new risk genes for Major Depression</p>
<p><a href="http://www.nature.com/nature/journal/v523/n7562/full/nature14659.html">Paper</a> <a href="http://www.nature.com/news/first-robust-genetic-links-to-depression-emerge-1.17979">News article</a></p>
<h4 id="1-july-2015">1 July 2015</h4>
<p>The first release of the <a href="http://www.haplotype-reference-consortium.org/">Haplotype Reference Consortium</a> dataset can now be accessed via imputation servers.</p>
<h4 id="29-may-2015">29 May 2015</h4>
<p>Victoria Hore won the prize for the best student talk at the Quantitative Genomics 2015 conference. Her talk was entitled ‘Sparse Bayesian latent factor decompositions for identifying trans-eQTLs.</p>
<h4 id="october-2014">October 2014</h4>
<p>We have done very well this year at the American Society of Human Genetics meeting in San Diego, with 3 members of the group being selected to give a talk about their research. Here are the details</p>
<p>Jonathan Marchini  Genetic network inference in studies of multiple phenotypes from related individuals.
Program Number: 24   Presentation Time: 02:15PM-02:30PM</p>
<p>Victoria Hore Sparse Bayesian latent factor decompositions for identifying trans-eQTLs.
Program Number: 262   Presentation Time: 12:15PM-12:30PM</p>
<p>Winni Kretzschmar A genotype likelihood based phasing and imputation method for massive sample sizes of low-coverage sequencing data.
Program Number: 150   Presentation Time: 10:30AM-10:45AM</p>
<p>Andy Dahl Mutiple-trait genomic selection and phenotype prediction.
Program Number: 1632S  Presentation Time: Sun, Oct 19, 2014, 5:00PM-6:00PM</p>
<h4 id="1-august-2014">1 August 2014</h4>
<p>The University of Oxford has made Jonathan Marchini a Professor of Statistical Genomics.</p>
<h4 id="16-june-2014">16 June 2014</h4>
<p>Our paper on calling genotypes and phasing has been published in <a href="http://www.nature.com/ncomms/2014/140606/ncomms4934/full/ncomms4934.html">Nature Communications</a></p>
<p>We have released a new version of <a href="https://mathgen.stats.ox.ac.uk/genetics_software/shapeit/shapeit.html">SHAPEIT2</a> that implements this functionality.</p>
<p>We have also released a new version of IMPUTE2 that fixes a bug in the panel merging functionality, and provided a minor update our phased version of the 1000 Genomes Phase 1 dataset.</p>
<h4 id="17-apr-2014">17 Apr 2014</h4>
<p>Jared O’Connell’s paper on a general method for phasing related samples has been published in <a href="https://journals.plos.org/plosgenetics/article?id=10.1371/journal.pgen.1004234">PLoS Genetics</a></p>
<p>The method is implemented within <a href="https://mathgen.stats.ox.ac.uk/genetics_software/shapeit/shapeit.html">SHAPEIT2</a> and also in a standalone package called duoHMM.</p>
<h4 id="24-mar-2014">24 Mar 2014</h4>
<p>We have just released a new version of <a href="https://mathgen.stats.ox.ac.uk/genetics_software/shapeit/shapeit.html">SHAPEIT2</a> (v2 r767) that implements functionality for (a) using phase information in sequencing reads to help phasing, and (b)phasing in general pedigrees. This work is associated with the following two publications</p>
<p><a href="http://www.cell.com/AJHG/abstract/S0002-9297%2813%2900415-1?utm_source=ECE001&amp;utm_campaign=&amp;utm_content=&amp;utm_medium=email&amp;bid=OTV0E5F:3IUI5">O. Delaneau, B. Howie, A. Cox, J-F. Zagury, J. Marchini (2013) Haplotype estimation using sequence reads. American Journal of Human Genetics 93 (4) 787-696</a></p>
<p><a href="https://journals.plos.org/plosgenetics/article?id=10.1371/journal.pgen.1004234">J. O’Connell, D. Gurdasani, O. Delaneau, et al. (2014) A general approach for haplotype phasing across the full spectrum of relatedness. PLoS Genetics</a></p>
<p>#### 18 Oct 2013
 We have done very well this year at the American Society of Human Genetics meeting in Boston, with 4 members of the group being selected to give a talk about their research. Here are the details</p>
<p>Jared O’Connell Haplotype phasing across the full spectrum of relatedness.
Platform presentation #192 Presentation Time: 03:15PM-03:30PM Session Location: Grand Ballroom West, Level 3, Convention Center</p>
<p>Marie Forest Simultaneous estimation of population size changes and splits times from population level resequencing studies
Platform presentation #268 Presentation Time: 08:00AM-08:15AM Session Location: Grand Ballroom West, Level 3, Convention Center</p>
<p>Jonathan Marchini A haplotype map derived from whole genome low-coverage sequencing of over 25,000 individuals.
Platform presentation #349 Presentation Time: 02:00PM-02:15PM Session Location: Grand Ballroom West, Level 3, Convention Center</p>
<p>Olivier Delaneau A new method for genotype calling and phasing for the 1000 Genomes Project leads to improved downstream imputation accuracy.
Platform presentation #352 Presentation Time: 02:45PM-03:00PM Session Location: Grand Ballroom West, Level 3, Convention Center</p>
<p>Valentina Iotchkova Bayesian sparse models of high-dimensional correlated traits in related and unrelated individuals.
Poster # 1777F Presentation Time: Fri, Oct 25, 2013, 10:30AM-11:30AM</p>
<p>Warren Kretzschmar  Phasing and imputation of 12,000 1x coverage whole genome sequenced Chinese women.
Poster # 1895W Presentation Time: Wed, Oct 23, 2013, 10:30AM-11:30AM</p>
<h4 id="3-oct-2013">3 Oct 2013</h4>
<p>Our paper on using phase information in sequencing reads for haplotype estimation has been published by the American Journal of Human Genetics</p>
<p><a href="http://www.cell.com/AJHG/abstract/S0002-9297%2813%2900415-1?utm_source=ECE001&amp;utm_campaign=&amp;utm_content=&amp;utm_medium=email&amp;bid=OTV0E5F:3IUI5">O. Delaneau, B. Howie, A. J. Cox, J-F. Zagury, J. Marchini (2013) Haplotype estimation using sequencing reads. AJHG 93 (4) 687-696.</a></p>
<h4 id="18-sep-2013">18 Sep 2013</h4>
<p>We have just released a new version of the Phase 1 haplotypes from the 1000 Genomes Project. Using a set of validation genotypes at SNP and biallelic indels we have been able to show that these haplotypes have lower genotype discordance and improved imputation performance into downstream GWAS samples, especially at low frequency. The haplotypes are available from the 1000GP website or the IMPUTE2 website.</p>
<h4 id="21-mar-2013">21 Mar 2013</h4>
<p>Our paper describing the genetic variation of short insertions and deletions using sequencing data from the 1000 Genomes Project has been published by <a href="http://genome.cshlp.org/content/early/2013/03/11/gr.148718.112">Genome Research</a></p>
<h4 id="29-dec-2012">29 Dec 2012</h4>
<p>Our paper describing the SHAPEIT (v2) method has been published in <a href="http://www.nature.com/nmeth/journal/v10/n1/full/nmeth.2307.html">Nature Methods</a></p>
<p>This paper presents a significantly new algorithm for haplotype estimation that combines together the best ideas from IMPUTE2 and SHAPEIT1. The new method clearly outperforms other existing methods in terms of phasing performance and is very competitive computationally.</p>
<h4 id="6-dec-2012">6 Dec 2012</h4>
<p>We have just released a new version of MULTIMIX (v1.1.0). This new version allows for missing data.</p>
<h4 id="4-dec-2012">4 Dec 2012</h4>
<p>We have just released a new versions of SHAPEIT (v2). The paper on this new version has now been accepted in Nature Methods and will appear soon.</p>
<h4 id="10-nov-2012">10 Nov 2012</h4>
<p>Olivier Delaneau has won a Charles Epstein Post-Doctoral Award (worth $2,000) at the ASHG conference in San Francisco for his work on SHAPEIT2. Well done Olivier! Androniki Menelaou made the final of the Pre-Doctoral competition (worth $1,000) but did not win a prize.</p>
<h4 id="7-nov-2012">7 Nov 2012</h4>
<p>Claire Churchhouse’s paper has been published in <a href="http://onlinelibrary.wiley.com/doi/10.1002/gepi.21692/abstract">Genetic Epidemiology</a></p>
<p>In this paper we develop a method for carrying out admixture deconvolution with more than two source ancestral populations. The method can handle both phased or unphased data from the source and/or sample populations. The method was used to analyze data from Phase 1 of the 1000 Genomes Project which was recently published in Nature. The MULTIMIX software implementing this approach will appear here soon.</p>
<h4 id="1-november-2012">1 November 2012</h4>
<p>Jonathan Marchini has been awarded a <a href="https://www.leverhulme.ac.uk/philip-leverhulme-prizes-2012">Philip Leverhulme Prize</a></p>
<h4 id="1-november-2012_1">1 November 2012</h4>
<p>The latest 1000 Genomes Project paper has been published in <a href="http://www.nature.com/nature/journal/v491/n7422/full/nature11632.html">Nature</a> today.</p>
<p>This is a big international collaboration that has sort to sequence a large number of individuals from around the world with the aim of mapping the majority of the genetic variation with frequency above 1%. Members of our group that contributed to the analysis were Olivier Delaneau, Claire Churchhouse, Jason Liu and myself. The project has been reported in the mainstream press.</p>
<h4 id="1-november-2012_2">1 November 2012</h4>
<p>The company Affymetrix has launched a new genotyping chip that was designed using our software IMPUTE2. They have issued a <a href="http://phx.corporate-ir.net/phoenix.zhtml?c=116408&amp;p=irol-newsArticle&amp;ID=1752709">press release giving the details</a></p>
<h4 id="23-october-2012">23 October 2012</h4>
<p>Androniki Menelaou’s paper has been published in <a href="http://bioinformatics.oxfordjournals.org/content/early/2012/10/23/bioinformatics.bts632.abstract?keytype=ref&amp;ijkey=G5i2fOHGfx7kzfc">Bioinformatics</a></p>
<p>In this paper we show that combining low-coverage sequence data with microarray genotyping on the same subjects can lead to high quality haplotype reference sets that produce a notable boost in downstream imputation accuracy. The MVNcall software implementing this approach will appear here soon.</p>
<h4 id="2-october-2012">2 October 2012</h4>
<p>Jared O’Connell won the poster prize at the recent Genomics of Common Diseases conference in Potomac, MD, USA.</p>
<h4 id="27-august-2012">27 August 2012</h4>
<p>We have done very well this year with accepted talks at the American Society of Human Genetics meeting. Olivier Delaneau and Androniki Menelaou have made the final of the post-doctoral and pre-doctoral Charles J. Epstein Trainee Awards</p>
<h4 id="24-august-2012">24 August 2012</h4>
<p>Claire Churchouse has succesfully passed her viva. Her thesis entitled ‘Bayesian Methods for Estimating Human Ancestry Using Whole Genome SNP Data’ was examined be Dr Lachlan Coin and Dr Simon Myers. Claire is now off to start a postdoc with David Althuler at the Broad Institute of Harvard and MIT. Good luck Claire!</p>
<h4 id="22-july-2012">22 July 2012</h4>
<p>Our new paper (with Bryan Howie as co-lead author) on pre-phasing based imputation has now appeared in <a href="http://www.nature.com/ng/journal/v44/n8/full/ng.2354.html">Nature Genetics</a></p>
<h4 id="20-july-2012">20 July 2012</h4>
<p>Jared O’Connell’s paper on a new method for calling genotypes from array and sequence data has now appeared in <a href="http://onlinelibrary.wiley.com/doi/10.1002/gepi.21657/abstract">Genetic Epidemiology</a></p>
<h4 id="29-june-2012">29 June 2012</h4>
<p>Androniki Menelaou has succesfully passed her viva. Her thesis entitled ‘LD-based SNP and genotype calling from next-generation sequencing reads” was examined be Dr Vincent Plagnol and Prof Gil McVean. Androniki is now off to start a postdoc with Paul de Bakker at UMC Utrecht. Good luck Androniki!</p>
<h4 id="30-april-2012">30 April 2012</h4>
<p>Victoria Hore has joined the group for her DTC project. Welcome Vicky!</p>
<h4 id="12-april-2012">12 April 2012</h4>
<p>Warren Kretzschmar has joined the group for his rotation project. Welcome Winni!</p>
<h4 id="15-march-2012">15 March 2012</h4>
<p>Olivier Delaneau’s paper on the SHAPEIT method has now appeared in <a href="http://www.nature.com/nmeth/journal/v9/n2/abs/nmeth.1785.html">Nature Methods</a></p></div>
        
        
    </div>

    
      <footer class="col-md-12 text-center">
          
          
            <hr>
            <p>
            <small>Documentation built with <a href="http://www.mkdocs.org/">MkDocs</a>.</small>
            </p>
          

          
          
      </footer>
    
    <script src="//ajax.googleapis.com/ajax/libs/jquery/1.12.4/jquery.min.js"></script>
    <script src="../js/bootstrap-3.0.3.min.js"></script>

    
    <script src="//cdn.jsdelivr.net/gh/highlightjs/cdn-release@9.18.0/build/highlight.min.js"></script>
        
    <script>hljs.initHighlightingOnLoad();</script>
    

    <script>var base_url = ".."</script>
    
    <script src="../js/base.js"></script>
    <script src="../search/main.js"></script>

    <div class="modal" id="mkdocs_search_modal" tabindex="-1" role="dialog" aria-labelledby="searchModalLabel" aria-hidden="true">
    <div class="modal-dialog modal-lg">
        <div class="modal-content">
            <div class="modal-header">
                <button type="button" class="close" data-dismiss="modal">
                    <span aria-hidden="true">&times;</span>
                    <span class="sr-only">Close</span>
                </button>
                <h4 class="modal-title" id="searchModalLabel">Search</h4>
            </div>
            <div class="modal-body">
                <p>
                    From here you can search these documents. Enter
                    your search terms below.
                </p>
                <form>
                    <div class="form-group">
                        <input type="text" class="form-control" placeholder="Search..." id="mkdocs-search-query" title="Type search term here">
                    </div>
                </form>
                <div id="mkdocs-search-results"></div>
            </div>
            <div class="modal-footer">
            </div>
        </div>
    </div>
</div><div class="modal" id="mkdocs_keyboard_modal" tabindex="-1" role="dialog" aria-labelledby="keyboardModalLabel" aria-hidden="true">
    <div class="modal-dialog">
        <div class="modal-content">
            <div class="modal-header">
                <h4 class="modal-title" id="keyboardModalLabel">Keyboard Shortcuts</h4>
                <button type="button" class="close" data-dismiss="modal"><span aria-hidden="true">&times;</span><span class="sr-only">Close</span></button>
            </div>
            <div class="modal-body">
              <table class="table">
                <thead>
                  <tr>
                    <th style="width: 20%;">Keys</th>
                    <th>Action</th>
                  </tr>
                </thead>
                <tbody>
                  <tr>
                    <td class="help shortcut"><kbd>?</kbd></td>
                    <td>Open this help</td>
                  </tr>
                  <tr>
                    <td class="next shortcut"><kbd>n</kbd></td>
                    <td>Next page</td>
                  </tr>
                  <tr>
                    <td class="prev shortcut"><kbd>p</kbd></td>
                    <td>Previous page</td>
                  </tr>
                  <tr>
                    <td class="search shortcut"><kbd>s</kbd></td>
                    <td>Search</td>
                  </tr>
                </tbody>
              </table>
            </div>
            <div class="modal-footer">
            </div>
        </div>
    </div>
</div>
    </body>

</html>
