<!DOCTYPE html>
<html lang="en">

<head>
    <meta charset="utf-8">
    <meta http-equiv="X-UA-Compatible" content="IE=edge">
    <meta name="viewport" content="width=device-width, initial-scale=1.0">
    <meta name="description" content="Jonathan Marchini research">
    <meta name="author" content="Jonathan Marchini">
    <link rel="canonical" href="https://sourcefoundry.org/cinder/pubs/">
    <link rel="shortcut icon" href="../img/favicon.ico">

    
    <title>Publications - jmarchini</title>
    

    <link rel="stylesheet" href="https://use.fontawesome.com/releases/v5.12.0/css/all.css">
    <link rel="stylesheet" href="https://use.fontawesome.com/releases/v5.12.0/css/v4-shims.css">
    <link rel="stylesheet" href="//cdn.jsdelivr.net/npm/hack-font@3.3.0/build/web/hack.min.css">
    <link href='//rsms.me/inter/inter.css' rel='stylesheet' type='text/css'>
    <link href='//fonts.googleapis.com/css?family=Open+Sans:300italic,400italic,700italic,400,300,600,700&subset=latin-ext,latin' rel='stylesheet' type='text/css'>
    <link href="../css/bootstrap-custom.min.css" rel="stylesheet">
    <link href="../css/base.min.css" rel="stylesheet">
    <link href="../css/cinder.min.css" rel="stylesheet">

    
        
        <link rel="stylesheet" href="//cdn.jsdelivr.net/gh/highlightjs/cdn-release@9.18.0/build/styles/github.min.css">
        
    

    <!-- HTML5 shim and Respond.js IE8 support of HTML5 elements and media queries -->
    <!--[if lt IE 9]>
            <script src="https://cdn.jsdelivr.net/npm/html5shiv@3.7.3/dist/html5shiv.min.js"></script>
            <script src="https://cdn.jsdelivr.net/npm/respond.js@1.4.2/dest/respond.min.js"></script>
        <![endif]-->

    

     
</head>

<body>

    <div class="navbar navbar-default navbar-fixed-top" role="navigation">
    <div class="container">

        <!-- Collapsed navigation -->
        <div class="navbar-header">
            <!-- Expander button -->
            <button type="button" class="navbar-toggle" data-toggle="collapse" data-target=".navbar-collapse">
                <span class="sr-only">Toggle navigation</span>
                <span class="icon-bar"></span>
                <span class="icon-bar"></span>
                <span class="icon-bar"></span>
            </button>
            

            <!-- Main title -->

            
              <a class="navbar-brand" href="..">jmarchini</a>
            
        </div>

        <!-- Expanded navigation -->
        <div class="navbar-collapse collapse">
                <!-- Main navigation -->
                <ul class="nav navbar-nav">
                
                
                    <li >
                        <a href="..">Home</a>
                    </li>
                
                
                
                    <li class="active">
                        <a href="./">Publications</a>
                    </li>
                
                
                
                    <li >
                        <a href="../projects/">Projects</a>
                    </li>
                
                
                
                    <li >
                        <a href="../software/">Software</a>
                    </li>
                
                
                
                    <li >
                        <a href="../people/">People</a>
                    </li>
                
                
                
                    <li >
                        <a href="../teaching/">Teaching</a>
                    </li>
                
                
                
                    <li >
                        <a href="../news/">News</a>
                    </li>
                
                
                </ul>

            <ul class="nav navbar-nav navbar-right">
                    <li>
                        <a href="#" data-toggle="modal" data-target="#mkdocs_search_modal">
                            <i class="fas fa-search"></i> Search
                        </a>
                    </li>
                    <li >
                        <a rel="prev" href="..">
                            <i class="fas fa-arrow-left"></i> Previous
                        </a>
                    </li>
                    <li >
                        <a rel="next" href="../projects/">
                            Next <i class="fas fa-arrow-right"></i>
                        </a>
                    </li>
            </ul>
        </div>
    </div>
</div>

    <div class="container">
        
        
        <div class="col-md-3"><div class="bs-sidebar hidden-print affix well" role="complementary">
    <ul class="nav bs-sidenav">
        <li class="first-level active"><a href="#publications">Publications</a></li>
            <li class="second-level"><a href="#2020">2020</a></li>
                
            <li class="second-level"><a href="#2019">2019</a></li>
                
            <li class="second-level"><a href="#2018">2018</a></li>
                
            <li class="second-level"><a href="#2017">2017</a></li>
                
            <li class="second-level"><a href="#2016">2016</a></li>
                
            <li class="second-level"><a href="#2015">2015</a></li>
                
            <li class="second-level"><a href="#2014">2014</a></li>
                
            <li class="second-level"><a href="#2013">2013</a></li>
                
            <li class="second-level"><a href="#2012">2012</a></li>
                
            <li class="second-level"><a href="#2011">2011</a></li>
                
            <li class="second-level"><a href="#2010">2010</a></li>
                
            <li class="second-level"><a href="#2009">2009</a></li>
                
            <li class="second-level"><a href="#2008">2008</a></li>
                
            <li class="second-level"><a href="#2007">2007</a></li>
                
            <li class="second-level"><a href="#2006">2006</a></li>
                
            <li class="second-level"><a href="#2005">2005</a></li>
                
            <li class="second-level"><a href="#2004">2004</a></li>
                
            <li class="second-level"><a href="#2003">2003</a></li>
                
            <li class="second-level"><a href="#2002">2002</a></li>
                
            <li class="second-level"><a href="#2001">2001</a></li>
                
            <li class="second-level"><a href="#2000">2000</a></li>
                
            <li class="second-level"><a href="#1997">1997</a></li>
                
    </ul>
</div></div>
        <div class="col-md-9" role="main">

<h2 id="publications">Publications</h2>
<p>h-index = 68<br />
total citations = 90,905<br />
<a href="https://scholar.google.com/citations?user=YdS6szoAAAAJ&amp;hl=en&amp;oi=ao">Google scholar page</a></p>
<h3 id="2020">2020</h3>
<ul>
<li>
<p>Matthew Kerin and Jonathan Marchini (2020) Non-linear randomized
Haseman-Elston regression for estimation of gene-environment
heritability. Bioinformatics   <a href="https://academic.oup.com/bioinformatics/advance-article/doi/10.1093/bioinformatics/btaa1079/6050717">[Journal]</a> <a href="https://www.biorxiv.org/content/10.1101/2020.05.18.098459v1">[bioRxiv]</a> <a href="https://mkerin.github.io/LEMMA/">[Software]</a></p>
</li>
<li>
<p>Matthew Kerin and Jonathan Marchini (2020) Inferring
Gene-by-Environment Interactions with a Bayesian Whole-Genome
Regression Model. American Journal of Human Genetics <a href="https://www.cell.com/ajhg/fulltext/S0002-9297(20)30277-9#">[Journal]</a>
<a href="https://mkerin.github.io/LEMMA/">[Software]</a></p>
</li>
<li>
<p>Simone Rubinacci, Olivier Delaneau and Jonathan Marchini (2020)
Genotype imputation using the Positional Burrows Wheeler
Transform. PLoS Genetics <a href="https://journals.plos.org/plosgenetics/article?id=10.1371/journal.pgen.1009049">[Journal]</a> <a href="https://www.dropbox.com/sh/mwnceyhir8yze2j/AADbzP6QuAFPrj0Z9_I1RSmla?dl=0">[Software]</a></p>
</li>
<li>
<p>Christopher Gill and Jonathan Marchini (2020) Four-Dimensional Sparse Bayesian Tensor Decomposition for Gene Expression Data <a href="https://www.biorxiv.org/content/10.1101/2020.11.30.403907v1">[bioRxiv]</a> <a href="https://github.com/marchinilab/SDA4D">[Software]</a></p>
</li>
</ul>
<h3 id="2019">2019</h3>
<ul>
<li>
<p>Delaneau et al. (2019) Accurate, scalable and integrative haplotype estimation. Nature Communications 10:5436 <a href="https://www.nature.com/articles/s41467-019-13225-y">[Journal]</a> <a href="https://odelaneau.github.io/shapeit4/">[Software]</a></p>
</li>
<li>
<p>Marchini J. L. (2019) Haplotype Estimation and Genotype Imputation. In Handbook of Statistical Genomics, 4th Edition <a href="https://www.wiley.com/en-gb/Handbook+of+Statistical+Genomics%2C+4th+Edition-p-9781119429142">[Link]</a></p>
</li>
<li>
<p>Marchini J. L. (2019). Discussion of ‘Gene hunting with knockoffs for hidden Markov models’. Biometrika 106, 27–8 <a href="https://academic.oup.com/biomet/article/106/1/27/5318366">[Link]</a></p>
</li>
<li>
<p>Jung et al. (2019) Unified single-cell analysis of testis gene regulation and pathology in 5 mouse strains eLife 2019;8:e43966 <a href="https://elifesciences.org/articles/43966">[Journal]</a></p>
</li>
</ul>
<h3 id="2018">2018</h3>
<ul>
<li>
<p>Elliott et al. (2018) Genome-wide association studies of brain structure and function in the UK Biobank. Nature 562:210–216 <a href="https://www.nature.com/articles/s41586-018-0571-7">[Journal]</a></p>
</li>
<li>
<p>Bycroft et al. (2017) Genome-wide genetic data on ~500,000 UK Biobank participants. Nature 562:203–209  <a href="https://www.nature.com/articles/s41586-018-0571-7">[Journal]</a> 
<a href="https://soundcloud.com/jenny-mills-362181607/uk-biobank-genetics-opens-the-door-to-a-new-era-of-health-research">[Podcast]</a> 
<a href="https://www.wired.co.uk/article/uk-biobank-genetic-studies-healthcare">[Wired article]</a> 
<a href="https://www.nature.com/articles/d41586-018-06950-9">[Nature Editorial]</a> 
<a href="https://www.nature.com/articles/d41586-018-06948-3">[Nature News and Views]</a>
<a href="https://www.news-medical.net/news/20181011/UK-Biobank-provides-wealth-of-information-for-further-genetic-studies.aspx">[News Medical Article]</a></p>
</li>
<li>
<p>Wiberg et al. (2018) Human Handedness: Genetics, Microtubules, Neuropsychiatric Diseases and Brain Language Areas <a href="https://www.biorxiv.org/content/10.1101/454660v1">[bioRxiv]</a></p>
</li>
<li>
<p>Gavin Band and Jonathan Marchini (2018) BGEN: a binary file format for imputed genotype and haplotype data <a href="https://www.biorxiv.org/content/early/2018/04/29/308296">[bioRxiv]</a> <a href="https://bitbucket.org/gavinband/bgen">[Software]</a></p>
</li>
<li>
<p>Wang et al. (2018). An imputation platform to enhance integration of rice genetic resources. Nature Communications 9:3519 <a href="https://www.nature.com/articles/s41467-018-05538-1">[Journal]</a> <a href="http://rice-impute.biotech.cornell.edu/">[Rice Imputation Server]</a></p>
</li>
</ul>
<h3 id="2017">2017</h3>
<ul>
<li>Justice et al. (2017) Genome-Wide Meta-Analysis of 241,258 Adults
Accounting for Smoking Behavior Identifies Novel Loci for Obesity
Traits. Nature Communications 8:14977 <a href="https://www.nature.com/articles/ncomms14977">[Journal]</a></li>
</ul>
<h3 id="2016">2016</h3>
<ul>
<li>
<p>The Haplotype Reference Consortium. A reference panel of 64,976 haplotypes for genotype imputation. Nature Genetics 10.1038/ng.3643 <a href="https://www.nature.com/articles/ng.3643">[Journal]</a></p>
</li>
<li>
<p>Victoria Hore, Ana Viñuela, Alfonso Buil, Julian Knight, Mark I McCarthy, Kerrin Small, Jonathan Marchini. Tensor decomposition for multi-tissue gene expression experiments. Nature Genetics 10.1038/ng.3624 <a href="https://www.nature.com/articles/ng.3624">[Journal]</a> <a href="https://www.dropbox.com/sh/chek4jkr28qnbrj/AABbHEVX_Ct70Dl54rKNKr6ra?dl=0">[Software]</a></p>
</li>
<li>
<p>Jared O’Connell, Kevin Sharp, Nick Shrine, Louise Wain, Ian Hall, Martin Tobin, Jean-Francois Zagury, Olivier Delaneau, Jonathan Marchini. Haplotype estimation for biobank scale datasets. Nature Genetics 10.1038/ng.3583 <a href="http://www.nature.com/articles/ng.3583">[Journal]</a> <a href="http://localhost:8000/shapeit3/">[Software]</a></p>
</li>
<li>
<p>Kevin Sharp, Olivier Delaneau, Warren Kretzschmar, Jonathan Marchini. Phasing for medical sequencing using rare variants and large haplotype reference panels. Bioinformatics doi:10.1093/bioinformatics/btw065 <a href="http://bioinformatics.oxfordjournals.org/content/early/2016/02/27/bioinformatics.btw065.abstract?keytype=ref&amp;ijkey=NicAjSqI9E6y0xG">[Journal]</a></p>
</li>
<li>
<p>Andrew Dahl, Valentina Iotchkova, Amelie Baud, Åsa Johansson, Ulf Gyllensten, Nicole Soranzo, Richard Mott, Andreas Kranis, Jonathan Marchini. A multiple phenotype imputation method for genetic studies. Nature Genetics doi:10.1038/ng.3513 <a href="http://www.nature.com/articles/ng.3513">[Journal]</a> <a href="https://www.dropbox.com/home/Oxford/software/software_webpages/phenix">[Software]</a></p>
</li>
<li>
<p>Astle et al. The Allelic Landscape of Human Blood Cell Trait Variation and Links to Common Complex Disease. Cell 10.1016/j.cell.2016.10.042 <a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC5300907/">[Journal]</a></p>
</li>
<li>
<p>Kevin Sharp, Wim Wiegerinck, Alejandro Arias-Vasquez, Barbara Franke, Jonathan Marchini, Cornelis A Albers,Hilbert J Kappen. Explaining Missing Heritability Using Gaussian Process Regression. <a href="http://biorxiv.org/content/early/2016/02/22/040576">[bioRxiv]</a></p>
</li>
</ul>
<h3 id="2015">2015</h3>
<ul>
<li>
<p>The 1000 Genomes Project Consortium (2015) A global reference for human genetic variation. Nature <a href="http://www.nature.com/nature/journal/v526/n7571/full/nature15393.html">[Journal]</a> <a href="http://www.nature.com/news/variety-of-life-1.18454">[Editorial]</a> <a href="http://www.nature.com/nature/journal/v526/n7571/full/526052a.html">[News and Views]</a></p>
</li>
<li>
<p>UK10K Project (2015) The UK10K project: rare variants in health and disease. Nature <a href="http://www.nature.com/nature/journal/vaop/ncurrent/full/nature14962.html">[Journal]</a></p>
</li>
<li>
<p>CONVERGE Consortium (2015) Sparse whole-genome sequencing identifies two loci for major depressive disorder. Nature 523 588-591 <a href="http://www.nature.com/nature/journal/v523/n7562/full/nature14659.html">[Journal]</a> <a href="http://www.nature.com/news/first-robust-genetic-links-to-depression-emerge-1.17979">[News article]</a></p>
</li>
<li>
<p>Jie Huang, Bryan Howie, Shane McCarthy, Yasin Memari, Klaudia Walter, Josine Min, Petr Danecek, Giovanni Malerba, Elisabetta Trabetti, Hou-Feng Zheng, UK10K Project, Giovanni Gambaro,J. Brent Richards, Richard Durbin, Nicholas Timpson, Jonathan Marchini &amp; Nicole Soranzo (2015) Improved imputation of low-frequency and rare variants using the UK10K haplotype reference panel. Nature Communications 6:8111 <a href="http://www.nature.com/ncomms/2015/150914/ncomms9111/full/ncomms9111.html">[Journal]</a></p>
</li>
<li>
<p>Wain, L. V. et al. (2015) Novel insights into the genetics of smoking behaviour, lung function and chronic obstructive pulmonary disease in UK Biobank. The Lancet Respiratory Medicine <a href="http://www.thelancet.com/journals/lanres/article/PIIS2213-2600%2815%2900283-0/abstract">[Journal]</a></p>
</li>
</ul>
<h3 id="2014">2014</h3>
<ul>
<li>
<p>J. O’Connell, D. Gurdasani, O. Delaneau, et al. (2014) A general approach for haplotype phasing across the full spectrum of
relatedness. PLoS Genetics doi: 10.1371/journal.pgen.1004234 <a href="http://www.plosgenetics.org/article/info:doi/10.1371/journal.pgen.1004234">[Journal]</a></p>
</li>
<li>
<p>O. Delaneau, J. Marchini, The 1000 Genomes Project Consortium (2014) Integrating sequence and array data to create an improved 1000 Genomes Project haplotype reference panel. Nature Communications <a href="http://www.nature.com/ncomms/2014/140606/ncomms4934/full/ncomms4934.html">[Journal]</a></p>
</li>
</ul>
<h3 id="2013">2013</h3>
<ul>
<li>
<p>Andy Dahl, Victoria Hore, Valentina Iotchkova, Jonathan Marchini (2013). Network inference in matrix-variate Gaussian models with non-independent noise. <a href="http://www.google.com/url?q=http%3A%2F%2Farxiv.org%2Fabs%2F1312.1622&amp;sa=D&amp;sntz=1&amp;usg=AFrqEzdbMyovXIex9zU3EarKyiTwl-eM3w">[arXiv]</a></p>
</li>
<li>
<p>O. Delaneau, B. Howie, A. Cox, J-F. Zagury, J. Marchini (2013) Haplotype estimation using sequence reads. American Journal of Human Genetics 93 (4) 787-696 <a href="http://www.cell.com/AJHG/abstract/S0002-9297%2813%2900415-1?utm_source=ECE001&amp;utm_campaign=&amp;utm_content=&amp;utm_medium=email&amp;bid=OTV0E5F:3IUI5">[Journal]</a></p>
</li>
<li>
<p>O. Delaneau, J-F. Zagury and J. Marchini (2013) Improved whole chromosome phasing for disease and population genetic studies. Nature Methods 10, 5-6. <a href="http://www.nature.com/nmeth/journal/v10/n1/full/nmeth.2307.html">[Journal]</a> <a href="https://mathgen.stats.ox.ac.uk/genetics_software/shapeit/shapeit.html">[Software]</a></p>
</li>
<li>
<p>A. Menelaou and J. Marchini (2013) Genotype calling and phasing using next-generation sequencing reads and a haplotype scaffold. Bioinformatics 29(1):84-91 <a href="http://bioinformatics.oxfordjournals.org/cgi/content/abstract/bts632?ijkey=G5i2fOHGfx7kzfc&amp;keytype=ref">[Journal]</a></p>
</li>
<li>
<p>C. Churchhouse and J. Marchini (2013) Multi-way admixture deconvolution using phased or unphased ancestral panels. Genetic Epidemiology 37(1):1-12 <a href="http://onlinelibrary.wiley.com/doi/10.1002/gepi.21692/abstract">[Journal]</a> <a href="https://jmarchini.org/multimix/">[Software]</a></p>
</li>
<li>
<p>S. Montgomery, D. Goode, E. Kvikstad, C. Albers, Z. Zhang, X. Mu, G. Ananda, B. Howie, K. Karczewski, K. Smith, V. Anaya, R. Richardson, J. Davis, D. MacArthur, A. Sidow, L. Duret, M. Gerstein, K. Markova, J. Marchini, G. McVean and G. Lunter (2013) The origin, evolution and functional impact of short insertion-deletion variants identified in 179 human genomes. Genome Research doi: 10.1101/gr.148718.112 <a href="https://genome.cshlp.org/content/early/2013/03/11/gr.148718.112">[Journal]</a></p>
</li>
<li>
<p>Wang J, Bansal AT, Martin M, Germer S, Benayed R, Essioux L, Lee JS, Begovich A, Hemmings A, Kenwright A, Taylor KE, Upmanyu R, Cutler P, Harari O, Marchini J, Criswell LA, Platt A. (2013) Genome-wide association analysis implicates the involvement of eight loci with response to tocilizumab for the treatment of rheumatoid arthritis. Pharmacogenomics J. 2013(3):235-41. doi: 10.1038/tpj.2012.8. <a href="http://www.ncbi.nlm.nih.gov/pubmed/22491018">[Journal]</a></p>
</li>
</ul>
<h3 id="2012">2012</h3>
<ul>
<li>
<p>The 1000 Genomes Project Consortium (2012) An integrated map of genetic variation from 1,092 human genomes. Nature 491, 56-65. <a href="http://www.nature.com/nature/journal/v491/n7422/full/nature11632.html">[Journal]</a></p>
</li>
<li>
<p>B. Howie, C. Fuchsberger, M. Stephens, J. Marchini, G. Abecasis (2012) Fast and accurate genotype imputation in genome-wide association studies through pre-phasing.  Nature Genetics. DOI: 10.1038/ng.2354 <a href="http://www.nature.com/ng/journal/vaop/ncurrent/full/ng.2354.html">[Journal]</a> <a href="https://mathgen.stats.ox.ac.uk/impute/impute_v2.html">[Software]</a></p>
</li>
<li>
<p>O. Delaneau, J. Marchini, JF. Zagury (2012) A linear complexity phasing method for thousands of genomes. Nature Methods doi:10.1038/nmeth.1785 <a href="http://www.nature.com/nmeth/journal/vaop/ncurrent/pdf/nmeth.1785.pdf">[Journal]</a> <a href="https://mathgen.stats.ox.ac.uk/genetics_software/shapeit/shapeit.html">[Software]</a></p>
</li>
<li>
<p>J. O’Connell, J. Marchini (2012) Joint Genotype Calling With Array and Sequence Data. Genetic Epidemiology. DOI: 10.1002/gepi.21657 <a href="http://onlinelibrary.wiley.com/doi/10.1002/gepi.21657/abstract">[Journal]</a></p>
</li>
</ul>
<h3 id="2011">2011</h3>
<ul>
<li>
<p>B. Howie, J. Marchini, M. Stephens (2011) Genotype Imputation with Thousands of Genomes. G3 : Genes, Genomes, Genetics. doi: 10.1534/g3.111.001198 <a href="http://www.g3journal.org/content/1/6/457.full">[Journal]</a> <a href="https://mathgen.stats.ox.ac.uk/impute/impute_v2.html">[Software]</a></p>
</li>
<li>
<p>N. Cardin, C. Holmes, WTCCC, P. Donnelly, J. Marchini (2011) Bayesian hierarchical mixture modeling to assign copy number from a targeted CNV array. Genetic Epidemiology DOI: 10.1002/gepi.20604 <a href="http://onlinelibrary.wiley.com/doi/10.1002/gepi.20604/abstract">[Journal]</a></p>
</li>
<li>
<p>Z. Su, J. Marchini, P. Donnelly (2011) HAPGEN2: simulation of multiple disease SNPs. Bioinformatics. doi: 10.1093/bioinformatics/btr341 <a href="http://bioinformatics.oxfordjournals.org/content/early/2011/06/08/bioinformatics.btr341.abstract?keytype=ref&amp;ijkey=p8zEzPePRYFEsjR">[Journal]</a> <a href="http://mathgen.stats.ox.ac.uk/genetics_software/hapgen/hapgen2.html">[Software]</a></p>
</li>
<li>
<p>T. Ferreira and J. Marchini (2011) Modeling interactions with known risk loci—a Bayesian model averaging approach. Annals of Human Genetics 75 (1) 1-9 <a href="http://onlinelibrary.wiley.com/doi/10.1111/j.1469-1809.2010.00618.x/abstract">[Journal]</a></p>
</li>
<li>
<p>M.Soler Artigas et al. (2011) Genome-wide association and large-scale follow up identifies 16 new loci influencing lung function. Nature Genetics 43 1082-90 <a href="http://www.nature.com/ng/journal/v43/n11/abs/ng.941.html">[Journal]</a></p>
</li>
<li>
<p>I. Tomlinson et al. (2011) Multiple common susceptibility variants near BMP pathway loci GREM1, BMP4, and BMP2 explain part of the missing heritability of colorectal cancer. PLoS Genet 7(6):e1002105 <a href="https://pubmed.ncbi.nlm.nih.gov/21655089/">[Journal]</a></p>
</li>
<li>
<p>L. Southam et al. (2011)  The effect of genome-wide association scan quality control on imputation outcome for common variants. Eur J Hum Genet 19(5):610-614 <a href="https://pubmed.ncbi.nlm.nih.gov/21267008/">[Journal]</a></p>
</li>
</ul>
<h3 id="2010">2010</h3>
<ul>
<li>
<p>J. Marchini and B. Howie (2010) Genotype imputation for genome-wide association studies. Nature Reviews Genetics <a href="http://www.nature.com/nrg/journal/vaop/ncurrent/abs/nrg2796.html">[Journal]</a></p>
</li>
<li>
<p>J. Liu et al. (2010) Meta-analysis and imputation refines the association of 15q25 with smoking quantity. Nature Genetics 42, 436–440 <a href="http://www.nature.com/ng/journal/v42/n5/abs/ng.572.html">[Journal]</a></p>
</li>
<li>
<p>Genome-wide association study of CNVs in 16,000 cases of eight common diseases and 3,000 shared controls. Wellcome Trust Case-Control Consortium. Nature. 2010;464;713-20. <a href="http://www.nature.com/nature/journal/v464/n7289/full/nature08979.html">[Journal]</a></p>
</li>
<li>
<p>The 1000 Genomes Consortium (2010) A map of human genome variation from population-scale sequencing. Nature 467, 1061–1073 <a href="http://www.1000genomes.org/sites/1000genomes.org/files/docs/nature09534.pdf">[Journal]</a></p>
</li>
</ul>
<h3 id="2009">2009</h3>
<ul>
<li>
<p>Z. Su, N. Cardin, WTCCC, P. Donnelly, J. Marchini (2009) A Bayesian method for detecting and characterizing allelic heterogeneity and boosting signals in genome-wide association studies. Statistical Science 24(4) 430-450. <a href="https://projecteuclid.org/journals/statistical-science/volume-24/issue-4/A-Bayesian-Method-for-Detecting-and-Characterizing-Allelic-Heterogeneity-and/10.1214/09-STS311.full">[Journal]</a> <a href="https://mathgen.stats.ox.ac.uk/genetics_software/genecluster/gc.html">[Software]</a></p>
</li>
<li>
<p>B. Howie, P. Donnelly, J. Marchini (2009) A Flexible and Accurate Genotype Imputation Method for the Next Generation of Genome-Wide Association Studies. PLoS Genetics  5(6): e1000529 <a href="http://www.plosgenetics.org/article/info%3Adoi%2F10.1371%2Fjournal.pgen.1000529">[Journal]</a></p>
</li>
<li>
<p>C. Spencer, Z. Su, P. Donnelly, J. Marchini (2009) Designing Genome-Wide Association Studies: Sample Size, Power, Imputation, and the Choice of Genotyping Chip. PLoS Genetics 5(5). <a href="http://www.plosgenetics.org/article/info%3Adoi%2F10.1371%2Fjournal.pgen.1000477">[Journal]</a> <a href="https://www.dropbox.com/s/nl76sh64jv0qa8h/GWApower_1.1.tar.gz?dl=0">[Software]</a></p>
</li>
<li>
<p>M. Jallow et al. (2009) Genome-wide and fine-resolution association analysis of malaria in West Africa. Nature Genetics 41, 657 – 665 <a href="https://www.nature.com/articles/ng.388">[Journal]</a></p>
</li>
<li>
<p>M. O’Donovan et al. (2009) Analysis of 10 independent samples provides evidence for association between schizophrenia and a SNP flanking fibroblast growth factor receptor 2. Mol Psychiatry. 14(1):30-6 <a href="https://pubmed.ncbi.nlm.nih.gov/18813210/">[Journal]</a></p>
</li>
</ul>
<h3 id="2008">2008</h3>
<ul>
<li>
<p>C. Barnes, V. Plagnol, T. Fitzgerald, R. Redon, J. Marchini, D. Clayton, M. Hurles (2009) A robust statistical method for case-control association testing with copy number variation. Nature Genetics 40(10):1245-52.<a href="https://www.nature.com/articles/ng.206">[Journal]</a></p>
</li>
<li>
<p>J. Barrett et al. (2008) Genome-wide association defines more than 30 distinct susceptibility loci for Crohn’s disease. Nat Genet 40(8):955-962 </p>
</li>
<li>
<p>R. Loos et al. (2008) Common variants near MC4R are associated with fat mass, weight and risk of obesity. Nat Genet 40(6):768-775. 24.<a href="https://www.nature.com/articles/ng.140">[Journal]</a></p>
</li>
</ul>
<h3 id="2007">2007</h3>
<ul>
<li>
<p>J. Marchini, B. Howie, S. Myers, G. McVean and P. Donnelly (2007) A new multipoint method for genome-wide association studies via imputation of genotypes. Nature Genetics 39 : 906-913 <a href="http://www.nature.com/ng/journal/v39/n7/full/ng2088.html">[Journal]</a><a href="http://www.nature.com/ng/journal/v39/n7/full/ng0707-815.html">[News and Views Article]</a></p>
</li>
<li>
<p>The Wellcome Trust Case Control Consortium (2007) Genome-wide association study of 14,000 cases of seven common diseases and 3,000 shared controls. Nature 447;661-78. PMID: 17554300 DOI: 10.1038/nature05911.<a href="https://www.nature.com/articles/nature05911">[Journal]</a></p>
</li>
<li>
<p>The International HapMap Consortium (2007) A second generation human haplotype map of over 3.1 million SNPs. Nature 449(7164):851 <a href="https://www.nature.com/articles/nature06258">[Journal]</a></p>
</li>
<li>
<p>Zeggini et al. (2007) Replication of genome-wide association signals in UK samples reveals risk loci for type 2 diabetes. Science. Jun 1;316(5829):1336-41. <a href="http://www.sciencemag.org/cgi/content/full/316/5829/1336">[Journal]</a></p>
</li>
<li>
<p>M. Newport et al. (2007) Association scan of 14,500 non-synonymous SNPs in four diseases identifies autoimmunity variants. Nature
Genetics 39(11):1329-1337.<a href="https://www.nature.com/articles/ng.2007.17">[Journal]</a></p>
</li>
</ul>
<h3 id="2006">2006</h3>
<ul>
<li>
<p>J. Marchini, D. Cutler, N. Patterson, M. Stephens, E. Eskin, E. Halperin, S. Lin, Z.S. Qin, H.M. Munro, G.R. Abecasis, P. Donnelly, and International HapMap Consortium (2006) A Comparison of Phasing Algorithms for Trios and Unrelated Individuals. American Journal of Human Genetics, 78 437-450 <a href="https://www.cell.com/ajhg/fulltext/S0002-9297(07)62383-0">[Journal]</a></p>
</li>
<li>
<p>P. De Bakker et al. (2006) A high-resolution HLA and SNP haplotype map for disease association studies in the extended human MHC. Nat Genet 38(10):1166-1172. <a href="https://www.nature.com/articles/ng1885">[Journal]</a></p>
</li>
<li>
<p>D. Evans, J. Marchini, A. Morris, J. Cardon (2006) Two-stage two-locus models in genome-wide association. PLoS Genet 2(9):e157 <a href="https://journals.plos.org/plosgenetics/article?id=10.1371/journal.pgen.0020157">[Journal]</a></p>
</li>
</ul>
<h3 id="2005">2005</h3>
<ul>
<li>
<p>J. Marchini, P. Donnelly and L. R Cardon (2005) Genome-wide strategies for detecting multiple loci influencing complex diseases. Nature Genetics, 37 413-417 <a href="https://www.nature.com/articles/ng1537">[Journal]</a> <a href="https://www.nature.com/articles/ng0405-337">[News and Views article]</a></p>
</li>
<li>
<p>The International HapMap Consortium. A haplotype map of the human genome. Nature 437, 1299-1320. 2005. <a href="https://www.nature.com/articles/nature04226">[Journal]</a></p>
</li>
</ul>
<h3 id="2004">2004</h3>
<ul>
<li>
<p>J. Marchini, L. Cardon, M. Phillips, and P. Donnelly (2004) The effects of human population structure on large genetic association studies. Nature Genetics 36 512 – 517. <a href="https://www.nature.com/articles/ng1337">[Journal]</a></p>
</li>
<li>
<p>The International HapMap Consortium (2004).  Integrating ethics and science in the International HapMap Project.  Nature Reviews Genetics, 5  467-475. <a href="https://www.nature.com/articles/nrg1351">[Journal]</a></p>
</li>
<li>
<p>J. Marchini and A. Presanis (2004) Comparing methods of analysing fMRI statistical parametric maps. NeuroImage, 22 (3) 1203-1213 <a href="https://www.sciencedirect.com/science/article/abs/pii/S105381190400165X">[Journal]</a></p>
</li>
</ul>
<h3 id="2003">2003</h3>
<ul>
<li>
<p>Marchini, J. L. and Cardon, L. R., (2003) Discussion of Nicholson et al. (2003) Assessing population differentiation and isolation from single-nucleotide polymorphism data. JRSS(B) 740-1.<a href="https://www.jstor.org/stable/3088810?seq=1">[Journal]</a></p>
</li>
<li>
<p>The International HapMap Consortium (2003). The international HapMap project. Nature 426 789-796. <a href="https://www.nature.com/articles/nature02168">[Journal]</a></p>
</li>
<li>
<p>Marchini, J. L. and Smith, S. M., (2003) On bias in the estimation of autocorrelations for fMRI voxel time series analysis. NeuroImage 18: 83-90 <a href="https://www.sciencedirect.com/science/article/abs/pii/S1053811902913212">[Journal]</a></p>
</li>
<li>
<p>Henry, C. J. K., Lightowler, H. J. and Marchini, J. (2003) Intra-individual variation inresting metabolic rate during the menstrual cycle. British Journal of Nutrition 89, 811-817 <a href="https://www.cambridge.org/core/journals/british-journal-of-nutrition/article/intraindividual-variation-in-resting-metabolic-rate-during-the-menstrual-cycle/80E30399BB2728B141AB537A8DD5353F">[Journal]</a></p>
</li>
</ul>
<h3 id="2002">2002</h3>
<ul>
<li>Marchini, J. L. (2002) AnalyzeFMRI: An R package for the exploration and analysis of MRI and fMRI datasets. R News 2 (1): 17-23</li>
</ul>
<h3 id="2001">2001</h3>
<ul>
<li>Banks, A. N., Marchini, J. L. and Guilford, T.C. (2001) Resolving navigational conflict: clock-shift experiments with homing pigeons. Proc. Royal Institute Navigation Conference, Oxford.</li>
</ul>
<h3 id="2000">2000</h3>
<ul>
<li>Marchini, J. L., and Ripley, B. D., (2000) A New Statistical Approach to Detecting Significant Activation in Functional MRI. NeuroImage 12: 366-380 <a href="https://www.sciencedirect.com/science/article/abs/pii/S1053811900906281?via%3Dihub">[Journal]</a></li>
</ul>
<h3 id="1997">1997</h3>
<ul>
<li>Crichton, N. J., Hinde, J. P., and Marchini, J. L., (1997) Models for Diagnosing Chest Pain: Is CART helpful? Statistics in Medicine 16: 717-727 <a href="https://pubmed.ncbi.nlm.nih.gov/9131760/">[Journal]</a></li>
</ul></div>
        
        
    </div>

    
      <footer class="col-md-12 text-center">
          
          
            <hr>
            <p>
            <small>Documentation built with <a href="http://www.mkdocs.org/">MkDocs</a>.</small>
            </p>
          

          
          
      </footer>
    
    <script src="//ajax.googleapis.com/ajax/libs/jquery/1.12.4/jquery.min.js"></script>
    <script src="../js/bootstrap-3.0.3.min.js"></script>

    
    <script src="//cdn.jsdelivr.net/gh/highlightjs/cdn-release@9.18.0/build/highlight.min.js"></script>
        
    <script>hljs.initHighlightingOnLoad();</script>
    

    <script>var base_url = ".."</script>
    
    <script src="../js/base.js"></script>
    <script src="../search/main.js"></script>

    <div class="modal" id="mkdocs_search_modal" tabindex="-1" role="dialog" aria-labelledby="searchModalLabel" aria-hidden="true">
    <div class="modal-dialog modal-lg">
        <div class="modal-content">
            <div class="modal-header">
                <button type="button" class="close" data-dismiss="modal">
                    <span aria-hidden="true">&times;</span>
                    <span class="sr-only">Close</span>
                </button>
                <h4 class="modal-title" id="searchModalLabel">Search</h4>
            </div>
            <div class="modal-body">
                <p>
                    From here you can search these documents. Enter
                    your search terms below.
                </p>
                <form>
                    <div class="form-group">
                        <input type="text" class="form-control" placeholder="Search..." id="mkdocs-search-query" title="Type search term here">
                    </div>
                </form>
                <div id="mkdocs-search-results"></div>
            </div>
            <div class="modal-footer">
            </div>
        </div>
    </div>
</div><div class="modal" id="mkdocs_keyboard_modal" tabindex="-1" role="dialog" aria-labelledby="keyboardModalLabel" aria-hidden="true">
    <div class="modal-dialog">
        <div class="modal-content">
            <div class="modal-header">
                <h4 class="modal-title" id="keyboardModalLabel">Keyboard Shortcuts</h4>
                <button type="button" class="close" data-dismiss="modal"><span aria-hidden="true">&times;</span><span class="sr-only">Close</span></button>
            </div>
            <div class="modal-body">
              <table class="table">
                <thead>
                  <tr>
                    <th style="width: 20%;">Keys</th>
                    <th>Action</th>
                  </tr>
                </thead>
                <tbody>
                  <tr>
                    <td class="help shortcut"><kbd>?</kbd></td>
                    <td>Open this help</td>
                  </tr>
                  <tr>
                    <td class="next shortcut"><kbd>n</kbd></td>
                    <td>Next page</td>
                  </tr>
                  <tr>
                    <td class="prev shortcut"><kbd>p</kbd></td>
                    <td>Previous page</td>
                  </tr>
                  <tr>
                    <td class="search shortcut"><kbd>s</kbd></td>
                    <td>Search</td>
                  </tr>
                </tbody>
              </table>
            </div>
            <div class="modal-footer">
            </div>
        </div>
    </div>
</div>
    </body>

</html>
